ChemistryLib/PhreeqcIOData/CreateAqueousSolution.cpp

```diff
@@ -65,10 +48,32 @@ AqueousSolution createAqueousSolution(BaseLib::ConfigTree const& config)
     }
     else
     {
-        means_of_adjusting_charge = MeansOfAdjustingCharge::Unspecified;
+        return MeansOfAdjustingCharge::Unspecified;
```

Comment:
Just a question; Is this valid and defined state or is it an error in general and a default will be chosen?

Reply:
This return is a common case.